src/disco/components/Addon/index.js

```diff
@@ -121,31 +123,58 @@ export class AddonBase extends React.Component {
   }
 
   getThemeImage() {
-    const { getBrowserThemeData, i18n, name, previewURL } = this.props;
-    if (this.props.type === ADDON_TYPE_THEME) {
+    const { type } = this.props;
+
+    if (ADDON_TYPE_THEMES.includes(type)) {
+      const {
+        getBrowserThemeData,
+        i18n,
+        name,
+        previewURL,
+        previews,
+      } = this.props;
+
+      let imageUrl =
+        previews && previews.length > 0 && previews[0].image_url
+          ? previews[0].image_url
+          : null;
+
+      if (!imageUrl && type === ADDON_TYPE_THEME) {
+        imageUrl = previewURL;
+      }
+
+      const headerImage = (
+        <img
+          alt={sprintf(i18n.gettext('Preview of %(name)s'), { name })}
+          className="Addon-theme-header-image"
+          src={imageUrl}
+        />
+      );
+
       /* eslint-disable jsx-a11y/href-no-hash, jsx-a11y/anchor-is-valid */
-      return (
+      return type === ADDON_TYPE_THEME ? (
         <a
-          href="#"
           className="theme-image"
           data-browsertheme={getBrowserThemeData()}
+          href="#"
           onClick={this.installTheme}
         >
-          <img
-            src={previewURL}
-            alt={sprintf(i18n.gettext('Preview of %(name)s'), { name })}
-          />
+          {headerImage}
+        </a>
+      ) : (
+        <a className="theme-image" href="#">
```

Comment:
Do we really need a `a` here? It has no link and no `onClick` handler.

Reply:
Yeh I guess it makes sense to move this to the functionality PR where it does have the its own onClick